warehouse/templates/accounts/profile.html

```diff
@@ -75,7 +76,7 @@ <h2>
         {% endif %}
       </h2>
       <div class="package-list">
-        {% if projects %}
+        {% if project_count %}
```

Comment:
I dug through this, and realized what was bothering me.

After this change, we confirm that there's actual projects, but the way the HTML is structured we end up with two `package-list` entries, and nothing differentiates them. Could we/should we combine the logic so that there's one instance of a `package-list` or at least provide some context to the div class that these are `active` vs `archived`? Or is that overthinking it for classes that don't have any CSS specific to them right now? (Ignore for a second that the CSS for `package-list` was removed a long time ago.)

This is not a blocker, but I wanted to note it in case someone thinks it's important for later.